package-lock.json

```diff
@@ -18668,7 +18668,7 @@
 		"app-root-dir": {
 			"version": "1.0.2",
 			"resolved": "https://registry.npmjs.org/app-root-dir/-/app-root-dir-1.0.2.tgz",
-			"integrity": "sha512-jlpIfsOoNoafl92Sz//64uQHGSyMrD2vYG5d8o2a4qGvyNCvXur7bzIsWtAC/6flI2RYAp3kv8rsfBtaLm7w0g==",
+			"integrity": "sha1-OBh+wt6nV3//Az/8sSFyaS/24Rg=",
```

Comment:
Not sure why this file is changed?

Reply:
I haven't made any changes to these files 🤔

Reply:
Yeah, it could be like that sometimes after running `npm install`. You can just revert the changes here by running `git checkout trunk -- package-lock.json` or something similar.